packages/toolkit/package.json

```diff
@@ -1462,12 +1462,12 @@
                 },
                 {
                     "command": "aws.ec2.linkToLaunch",
-                    "group": "0@1",
+                    "group": "0@0",
```

Comment:
Does this avoid the need for the command name changes? We probably should stick with `Open Terminal` if possible, since we use that in the command-palette names, and for ECS, CodeCatalyst, etc.

Reply:
Yea it does, I can swap it back.